src/commcare_cloud/ansible/roles/kafka/templates/server.properties.j2

```diff
@@ -117,3 +117,9 @@ zookeeper.connect={% for host in groups['zookeeper'] %}{{ host }}:{{ zookeeper_c
 
 # Timeout in ms for connecting to zookeeper
 zookeeper.connection.timeout.ms=1000000
+{% if kafka_version == "2.4.1" %}
+inter.broker.protocol.version=2.4
+log.message.format.version=2.4
```

Comment:
I think these need to be configurable using a variable since initially they would be set to `0.8.2.2` and only changed to 2.4 after the upgrade is complete.

Reply:
i don't know how we are going to check the health of the system with the old value and how to trigger adding the new value